taxonomy-editor-frontend/src/pages/startproject/index.jsx

```diff
@@ -0,0 +1,100 @@
+import { Typography, Box, Grid, TextField, Stack, Autocomplete, Snackbar, Alert } from "@mui/material";
+import LoadingButton from '@mui/lab/LoadingButton';
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { TAXONOMY_NAMES } from "../../constants";
+import { createBaseURL } from "../editentry/createURL";
+
+const StartProject = () => {
+    const [branchName, setBranchName] = useState("")
+    const [taxonomyName, setTaxonomyName] = useState("additives")
+    const [description, setDescription] = useState("");
+    const [loading, setLoading] = useState(false); 
+    const [open, setOpen] = useState(false);
+    const [errorMessage, setErrorMessage] = useState("")
+    const navigate = useNavigate();
+    
+    function handleSubmit() {
+        const url = createBaseURL(taxonomyName, branchName)
+        setLoading(true);
+        const dataToBeSent = {'description' : description};
+        fetch(url+'import', {
+            method : 'POST',
+            headers: {"Content-Type" : "application/json"},
+            body: JSON.stringify(dataToBeSent)
+        }).then((response) => {
+            if (!response.ok) {
+                return response.json();
+            }
+            navigate(`/${taxonomyName}/${branchName}/entry`)
+        }).then((responseBody) => {
+            setErrorMessage(responseBody.detail)
```

Comment:
are these errors user friendly ?

Reply:
Yes, the errors returned by the backend are user friendly.